src/main/java/com/jagrosh/jdautilities/utils/SafeIdUtil.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2017 Kaidan Gustave
```

Comment:
Instead of the copyright you can put your name as an `@author` label in the class-level documentation.